crypto: se050: update policies

ldts opened the pull request:
Certain devices (depending on the applet) will use the common policies to for the object's read/write permissions.

Make sure both are supported.

Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>

<!--
    If you are new to submitting pull requests to OP-TEE, then please have a
    look at the list below and tick them off before submitting the pull request.

    1. Read our contribution guidelines:
         https://optee.readthedocs.io/en/latest/general/contribute.html

    2. Read the contribution section in Notice.md and pay extra attention to the
       "Developer Certificate of Origin" in the contribution guidelines.

    3. You should run checkpatch preferably before submitting the pull request.

    4. When everything has been reviewed, you will need to squash, rebase and
       add tags like `Reviewed-by`, `Acked-by`, `Tested-by` etc. More details
       about this can also be found on the link provided above.

    NOTE: This comment will not be shown in the pull request, so no harm keeping
    it, but feel free to remove it if you like.
-->

jenswi-linaro reviewed (commented):
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

ldts commented:
updated tag and fixed wording error on the commit. 

this can be merged now - it will make sure devices with newer applets are supported (otherwise access to their keys will be forbidden by the policy)
